### PR TITLE
Add annotations to code docs

### PR DESCRIPTION
**Pull request by keanulee**
Ref #196

### CODE REVIEW

File: app-header/demo/pesto.html [hunk 1]
```diff
@@ -1,97 +0,0 @@
-<!--
```
Comment (keanulee) [inline app-header/demo/pesto.html]: This demo's broken, and there's already the Pesto template, so I'm going to just remove it.

File: app-scrollpos-control/app-scrollpos-control.html [hunk 2]
```diff
@@ -46,7 +46,9 @@
 </app-drawer-layout>
 ```
 
-@demo app-scrollpos-control/demo/index.html
+@group App Elements
+@element app-scrollpos-control
+@demo app-scrollpos-control/demo/index.html Demo
```
Comment (keanulee) [inline app-scrollpos-control/app-scrollpos-control.html]: @blasten Do I need the "Demo" description here?

File: app-drawer/app-drawer.html [hunk 3]
```diff
@@ -45,8 +45,10 @@
 `--app-drawer-content-container` | Mixin for the drawer content container | {}
 `--app-drawer-scrim-background`  | Background for the scrim               | rgba(0, 0, 0, 0.5)
 
-@demo app-drawer/demo/demo1.html
-@demo app-drawer/demo/demo2.html
+@group App Elements
+@element app-drawer
+@demo app-drawer/demo/demo1.html Simple Left Drawer
```
Comment (blasten) [inline app-drawer/app-drawer.html]: consider renaming the file name.

**Comment by blasten** [comment]
@keanulee it looks good to me, the only change I'd suggest is renaming the demo files to match the description, so we can close https://github.com/PolymerLabs/app-layout/issues/194

File: app-drawer/app-drawer.html [hunk 4]
```diff
@@ -47,8 +47,8 @@
 
 @group App Elements
 @element app-drawer
-@demo app-drawer/demo/demo1.html Simple Left Drawer
-@demo app-drawer/demo/demo2.html Right Drawer with Icons
+@demo app-drawer/demo/index.html Simple Left Drawer
```
Comment (blasten) [inline app-drawer/app-drawer.html]: `app-drawer/demo/index.html` -> `app-drawer/demo/left-drawer.html` Then we can have an index.html that links to both demos. That way, people that clicked on "demo" from the standalone version of iron-component-page can access all the demos.

File: app-drawer/app-drawer.html [hunk 5]
```diff
@@ -45,8 +45,10 @@
 `--app-drawer-content-container` | Mixin for the drawer content container | {}
 `--app-drawer-scrim-background`  | Background for the scrim               | rgba(0, 0, 0, 0.5)
 
-@demo app-drawer/demo/demo1.html
-@demo app-drawer/demo/demo2.html
+@group App Elements
+@element app-drawer
+@demo app-drawer/demo/index.html Simple Left Drawer
```
Comment (blasten) [inline app-drawer/app-drawer.html]: `app-drawer/demo/index.html` -> `app-drawer/demo/left-drawer.html`. Then we can have an index.html that links to all the demos. That way, people who clicked "demo" from the standalone version of iron-component-page can access all the demos.

Comment (keanulee) [inline app-drawer/app-drawer.html]: Do we have a sample of this index page? I just saw that iron-list had one index.html and a bunch of other samples https://github.com/PolymerElements/iron-list/blob/master/iron-list.html#L142-L149

Comment (blasten) [inline app-drawer/app-drawer.html]: True, that is bad.

**Comment by blasten** [comment]
LGTM! thanks

**Pull request merged**